crates/sui-indexer-alt/src/config.rs

```diff
@@ -246,7 +248,7 @@ impl ConcurrentLayer {
                 (None, _) | (_, None) => None,
                 (Some(pruner), Some(base)) => Some(pruner.finish(base)),
             },
-            checkpoint_lag: self.checkpoint_lag.or(base.checkpoint_lag),
+            checkpoint_lag: base.checkpoint_lag,
```

Comment:
Because `ConcurrentLayer` does not have the `checkpoint_lag` field.

Reply:
But you added `checkpoint_lag` to `ConcurrentConfig` right? Why not add it to `ConcurrentLayer`? Even if it's unlikely that someone would want to set a checkpoint lag for all concurrent layers, it seems confusing to selectively offer the override logic -- it breaks the mental model for people who are interacting with this mainly be reading the configs.

Reply:
An alternative is to remove this field entirely even from ConcurrentConfig.
The primary intention is indeed to make sure that users cannot specify a global lag for all concurrent pipelines, since it does not make sense. This field comes from the consistency layer, instead of concurrent layer.